dev_tools/conf/.pylintrc

```diff
@@ -1,4 +1,5 @@
 [MASTER]
+load-plugins=pylint.extensions.docparams,pylint.extensions.docstyle
```

Comment:
interestingly I learned recently that pylint has a parallel mode (multiple cpus running lint), and using it you cannot use plugins :(

Reply:
Interesting, that sucks a bit. Our linter takes 2.4 minutes on my Mac today - and with that it's still well below the other CI tests. I think we can fit this in. If it becomes too lengthy, we can separate the two modes - quick'n'dirty and full-with-docs. The latter could run nightly and the former could be the CI version.